jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -9,6 +9,7 @@ $dark-grey: #9fa7aa;
 
 // Status colours.
 $uncommitted: #19b6ee;
+$error: #df382c;
```

Comment:
`$error` variable should already to accessible via the theme: 
https://github.com/ubuntudesign/cloud-vanilla-theme/blob/master/scss/_global-settings.scss#L15